hal/src/nRF52840/ble_hal_impl.h

```diff
@@ -79,7 +79,9 @@
 #define BLE_MAX_ADV_DATA_LEN                        BLE_GAP_ADV_SET_DATA_SIZE_MAX
 
 /* Maximum length of the buffer to store scan report data */
-#define BLE_MAX_SCAN_REPORT_BUF_LEN                 BLE_GAP_SCAN_BUFFER_EXTENDED_MAX_SUPPORTED  /* Must support extended length for CODED_PHY scanning */
+#define BLE_MAX_ADV_DATA_LEN_EXT                    BLE_GAP_SCAN_BUFFER_EXTENDED_MAX_SUPPORTED  /* Must support extended length for CODED_PHY scanning */
```

Comment:
We should keep `BLE_MAX_SCAN_REPORT_BUF_LEN` as-is and use it for scan buffers. And add:
```
#define BLE_MAX_ADV_DATA_LEN_EXT BLE_GAP_ADV_SET_DATA_SIZE_EXTENDED_MAX_SUPPORTED
#define BLE_MAX_ADV_DATA_LEN_EXT_CONNECTABLE BLE_GAP_ADV_SET_DATA_SIZE_EXTENDED_CONNECTABLE_MAX_SUPPORTED
```
Let's use `BLE_MAX_ADV_DATA_LEN_EXT_CONNECTABLE` for advertising data buffers for now and leave `BLE_MAX_ADV_DATA_LEN_EXT` alone.

Reply:
👍